.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "python.formatting.provider": "black"
+}
```

Comment:
Answer to your question:
https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control

Reply:
Seems like there are the usual diversity of opinions on stack overflow but it appears that it is permissible in this case. I will leave it in the repo for now